solidity/contracts/bridge/Bridge.sol

```diff
@@ -119,6 +119,8 @@ contract Bridge is Governable, EcdsaWalletOwner {
 
     event MovingFundsTimedOut(bytes20 walletPubKeyHash);
 
+    event MovingFundsBelowDustReported(bytes20 walletPubKeyHash);
```

Comment:
I would call it just `MovingFundsBelowDust` or `MovingFundsWentBelowDust` (the first one preferred). `*Reported* feels to me like we should expect a second part of the action.